Docker/Production/Dockerfile

```diff
@@ -22,18 +21,16 @@ WORKDIR /app/Source/Server
 RUN dotnet restore
 RUN dotnet publish -c ${CONFIGURATION} -o out
 
-# # Web Build
-# FROM node:13.12 AS web-build
-# WORKDIR /app
+# Web Build
+FROM node:14.2.0 AS web-build
+WORKDIR /app
 
-# COPY tslint.json ./
-# COPY tsconfig.settings.json ./
-# COPY Source/ManagementUI ./Source/ManagementUI
+COPY Source/Management.Web ./Source/Management.Web
 
-# WORKDIR /app/Source/ManagementUI
+WORKDIR /app/Source/Management.Web
 
-# RUN yarn
-# RUN yarn build
+RUN yarn
+RUN yarn build
```

Comment:
Do we want this UI thing by default in the production release - or maybe just the development+debug releases?

Haven't thought a lot about it - but I'm just wondering if there are some resource overheads or potential security problems with it on in the production version?

Reply:
We could do this for Dev only for now. The UI is one thing, but the underlying GraphQL endpoint would still be there. One way would be to feature toggle them both; have it in the image but you can basically opt in/out which would for Startup.cs mean that the GraphQL endpoints wouldn’t be added - nor the StaticFiles middleware.